src/math/src/sha512.cairo

```diff
@@ -8,6 +8,23 @@ pub const SHA512_LEN: usize = 64;
 
 pub const U64_BIT_NUM: u64 = 64;
 
+// Powers of two to avoid recomputing
+pub const TWO_56: u64 = 0x100000000000000;
+pub const TWO_48: u64 = 0x1000000000000;
+pub const TWO_40: u64 = 0x10000000000;
+pub const TWO_32: u64 = 0x100000000;
+pub const TWO_24: u64 = 0x1000000;
+pub const TWO_16: u64 = 0x10000;
+pub const TWO_8: u64 = 0x100;
+pub const TWO_4: u64 = 0x10;
+pub const TWO_2: u64 = 0x4;
+pub const TWO_1: u64 = 0x2;
+pub const TWO_0: u64 = 0x1;
```

Comment:
Are those names descriptive enough?
Shouldn't `TWO_POW_<n>` be better?

As they are all public it is important

Reply:
Yes, totally.
Fixed!